torch/distributed/pipelining/schedules.py

```diff
@@ -837,13 +851,13 @@ def _reshard(stage_index: int):
         # Unclear what the best policy is for eviction, but we can maintain order so we do
         evict = list(filter(lambda s: s not in next_n, active_stages))
 
-        logger.debug(
-            "_add_unshard_reshard Step %d active: %s fetch %s, evict %s",
-            i,
-            active_stages,
-            fetch,
-            evict,
-        )
+        # logger.debug(
```

Comment:
non-accidental. i felt it was too verbose even for when i enable + logging and want to look at other verbose logs.  But i felt like its worth leaving in in case i was ever debugging this pass again.